padertorch/io.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+
+from paderbox.io.new_subdir import get_new_subdir
+
+
+def get_new_storage_dir(
+        experiment_name,
+        *,
+        id_naming='index',
+        mkdir=True,
+        prefix=None,
+        suffix=None,
+        consider_mpi=False,
+        dry_run=False,
+):
+    """
+    This is a wrapper around `paderbox.io.new_subdir.get_new_subdir`.
+    
```

Comment:
Copy the parameter docs from paderbox? I'm not sure if referencing it is enough. Especially the mpi warning is important

Reply:
Done